packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -201,10 +249,10 @@ class OpPerfTelemetry {
                 category,
                 pingLatency: this.pingLatency,
                 msnDistance: this.deltaManager.lastSequenceNumber - this.deltaManager.minimumSequenceNumber,
-                durationInboundQueue: this.durationSittingInInboundQueue,
+                ...this.opPerfData,
```

Comment:
Can you please check in debugger (using samples) that
- all numbers make sense (maybe inject delays here and there to validate we are recording them properly)
- that numbers are integers (if not, we might way to use TelemetryLogger.formatTick to make it show up nicer in telemetry).
-

Reply:
Yes, I've been inspecting them in the debugger and they look nice (integers).